### PR TITLE
Add instructions on how to update the javadoc

### PR DESCRIPTION
**Pull request by bogdandrutu**
Fixes https://github.com/census-instrumentation/opencensus-java/issues/819

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -188,6 +188,28 @@ Central (the staging repository will be destroyed in the process). You can see
...
+
+Go through PR review and push the gh-pages branch to GitHub:
+
+```bash
+$ git checkout gh-pages
+$ git merge --ff-only update-javadoc
+$ git push upstream gh-pages
+```
+
+Check that the javadoc is updated [here](http://opencensus.io/opencensus-java).
```
Comment (songy23) [inline RELEASING.md]: Should this be https://github.com/census-instrumentation/opencensus-java/tree/gh-pages ?

File: RELEASING.md [hunk 2]
```diff
@@ -188,6 +188,28 @@ Central (the staging repository will be destroyed in the process). You can see
 the complete process for releasing to Maven Central on the [OSSRH
 site](http://central.sonatype.org/pages/releasing-the-deployment.html).
 
+### Update JavaDoc
+
+```bash
+$ git checkout -b update-javadoc gh-pages
```
Comment (sebright) [inline RELEASING.md]: "upstream/gh-pages" ?

File: RELEASING.md [hunk 3]
```diff
@@ -188,6 +188,28 @@ Central (the staging repository will be destroyed in the process). You can see
...
+$ rm -fr *
+$ wget -O opencensus-api-javadoc.jar “http://central.maven.org/maven2/io/opencensus/opencensus-api/$MAJOR.$MINOR.$PATCH/opencensus-api-$MAJOR.$MINOR.$PATCH-javadoc.jar”
+$ unzip -d . opencensus-api-javadoc.jar
+$ rm opencensus-api-javadoc.jar
+$ rm -r META-INF/
+$ git commit -a -m "Javadoc for $MAJOR.$MINOR.$PATCH"
```
Comment (sebright) [inline RELEASING.md]: This should probably include "git add -A" for new files.

File: RELEASING.md (hunk 2 above)
Comment (HailongWen) [reply RELEASING.md]: Agree. "gh-pages" is not recognized and need to explicitly specify remote.

File: RELEASING.md (hunk 1 above)
Comment (HailongWen) [reply RELEASING.md]: IMHO this is a black box check and it should be fine.

File: RELEASING.md [hunk 4]
```diff
@@ -188,6 +188,28 @@ Central (the staging repository will be destroyed in the process). You can see
 the complete process for releasing to Maven Central on the [OSSRH
 site](http://central.sonatype.org/pages/releasing-the-deployment.html).
 
+### Update JavaDoc
+
+```bash
+$ git checkout -b update-javadoc gh-pages
+$ rm -fr *
+$ wget -O opencensus-api-javadoc.jar “http://central.maven.org/maven2/io/opencensus/opencensus-api/$MAJOR.$MINOR.$PATCH/opencensus-api-$MAJOR.$MINOR.$PATCH-javadoc.jar”
```
Comment (songy23) [inline RELEASING.md]: Remove " before and after the url?

**Comment by bogdandrutu** [comment]
It is easier to use [javadoc.io](https://www.javadoc.io/doc/io.opencensus/opencensus-api/0.12.0) than gh-pages.

Close this PR and disable gh-pages.

**Pull request closed without merging**